app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt

```diff
@@ -15,6 +15,17 @@ import io.github.droidkaigi.confsched2018.model.toReadableDateTimeString
 import io.github.droidkaigi.confsched2018.model.toReadableTimeString
 import java.util.regex.Pattern
 
+@BindingAdapter(value = ["bind:dayNumber", "bind:suffix"])
+fun TextView.setDayText(dayNumber: Int?, suffix: String?) {
+    dayNumber ?: return
+    suffix ?: return
+    text = context.getString(
+            R.string.day_number_format,
+            dayNumber,
+            suffix
```

Comment:
I think it's better to separate TextViews for day number and slash.
What do you think?

Reply:
A [similar code](https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/res/layout/item_horizontal_session.xml#L53) has `/` that defined as prefix. so I copied that pattern.

but I agree your suggestion. I'll fix it as well!